test/Interop/Cxx/class/Inputs/module.modulemap

```diff
@@ -141,3 +141,8 @@ module CopyMoveAssignment {
   header "copy-move-assignment.h"
   requires cplusplus
 }
+
+module NonTrivialClasses {
```

Comment:
Should we call this `module Closures`, since closures are the special thing in this test module?